packages/runner/src/telemetry.ts

```diff
@@ -25,6 +23,40 @@ export type RuntimeTelemetryMarker = {
   type: "scheduler.invocation";
   handler: EventHandler | AnnotatedEventHandler;
   error?: string;
+} | {
+  type: "storage.push.start";
+  id: string;
+  operation: string;
+} | {
+  type: "storage.push.complete";
```

Comment:
we have `storage.x.{start|complete|error}` for telemetry types, the `.complete` markers always will have `success: true`. Consider `storage.x.start` and `storage.x.complete` with `error?: string` that can represent both success and fail cases